flow/flow.go

```diff
@@ -17,73 +17,73 @@ type Ckflow C.kflow
 const MAX_CUSTOM_STR_LEN = 384
 
 type Flow struct {
-	TimestampNano      int64
-	DstAs              uint32
-	DstGeo             uint32
-	DstMac             uint32 // IGNORED - use DstEthMac
-	HeaderLen          uint32
+	SrcBgpCommunity    string
```

Comment:
Let's add a top level comment on this struct to explicitly mentioning the ordering is done with struct padding in mind having 64-bit boundaries. This will help make it known that we've done this intentionally in an explicit way to avoid possible regressions of moving things around and so that we know when adding new fields that we should be conscious of the field ordering.